contrib/babelfishpg_tsql/src/pl_handler.c

```diff
@@ -169,6 +169,15 @@ static void bbf_ExecDropStmt(DropStmt *stmt);
 static int isolation_to_int(char *isolation_level);
 static void bbf_set_tran_isolation(char *new_isolation_level_str);
 
+typedef struct {
+	int oid;
+	char *alias;
```

Comment:
Did you consider the alias is not a normal collation ? Meaning not a normal char .

Reply:
Adding test cases for special languages